### PR TITLE
Use arguments to pass Ollama API endpoint and model. Add support for JPEG format

### PR DESCRIPTION
**Pull request by nikolicjakov**
This change modifies the main.ts file to use arguments to pass the Ollama API endpoint and model as command line arguments. Previously, these were hardcoded into the file. This change allows for more flexibility when running the script, as the API endpoint and model can now be specified each time the script is run. Additionally, a new directory called ai-renamed is created if it doesn't exist already and renamed images are created there. Also support for JPEG images is added.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,11 @@
-## [0.0.4](https://github.com/technovangelist/airenamer/compare/v0.0.3...v0.0.4) (2024-01-06)
+## 0.0.6 (2024-01-14)
```
Comment (technovangelist) [inline CHANGELOG.md]: remove changes to changelog. this is updated automatically

File: main.ts [hunk 2]
```diff
@@ -1,15 +1,21 @@
 import { Base64 } from "https://deno.land/x/bb64@1.1.0/mod.ts";
+import { existsSync } from "https://deno.land/std@0.212.0/fs/exists.ts";
```
Comment (technovangelist) [inline main.ts:2]: not sure why you are adding this

File: package.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "version": "0.0.4"
+  "version": "0.0.6"
```
Comment (technovangelist) [inline package.json]: remove this. it is automatically updated with the github action

File: readme.md [hunk 4]
```diff
@@ -18,3 +18,21 @@ A simple executable that renames image files based on keywords that describe the
 Navigate to a folder with some images in it and run `airenamer`.
 
 Make sure you have a backup first. It only copies the files, but I'm not responsible for any data loss.
+This is useful when you have Ollama running on remote host and you want to use different model.
```
Comment (technovangelist) [inline readme.md]: what is 'this'. not sure how it relates to the previous sentence.

File: readme.md [hunk 5]
```diff
@@ -18,3 +18,21 @@ A simple executable that renames image files based on keywords that describe the
...
+You can override the default Ollama API endpoint address and model by passing in arguments when calling the tool.
+
+| Argument   | Description                                              | Default value                       |
+| ---------- | -------------------------------------------------------- | -----------------------------      |
+| `base_url` | The base URL for the Ollama API endpoint.                | `http://localhost:11434`           |
+| `model`    | The Ollama model to use for generating image descriptions.| `llava:13b`                        |
+
+To override default values when calling the tool, you can use the following example:
+
+```
```
Comment (technovangelist) [inline readme.md]: if i just want to change the model and not the url, will that work? I think i would prefer to use an environment variable.

**Review by technovangelist — COMMENTED** [review]
Wow, thanks for submitting this. Just a few changes requested.

File: readme.md (hunk 5 above)
Comment (nikolicjakov) [reply readme.md]: I have changed it to use environment variables instead of arguments.

File: readme.md (hunk 4 above)
Comment (nikolicjakov) [reply readme.md]: This was a typo and now it is corrected in latest update.

File: package.json (hunk 3 above)
Comment (nikolicjakov) [reply package.json]: Removed. Sorry for that but i do not have experience with github actions since i use GitLab CI/CD.

File: main.ts (hunk 2 above)
Comment (nikolicjakov) [reply main.ts:2]: This is used to check if newDir exists or not (folder where renamed images are going to be stored). If it does not exist it is going to be created if it does it is going to be skipped.

File: CHANGELOG.md (hunk 1 above)
Comment (nikolicjakov) [reply CHANGELOG.md]: Removed. Sorry for that but i do not have experience with github actions since i use GitLab CI/CD.

**Pull request closed without merging**